tianshou/env/venvs.py

```diff
@@ -371,8 +371,13 @@ class DummyVectorEnv(BaseVectorEnv):
         Please refer to :class:`~tianshou.env.BaseVectorEnv` for other APIs' usage.
     """
 
-    def __init__(self, env_fns: Sequence[Callable[[], ENV_TYPE]], **kwargs: Any) -> None:
-        super().__init__(env_fns, DummyEnvWorker, **kwargs)
+    def __init__(
+        self,
+        env_fns: Sequence[Callable[[], ENV_TYPE]],
+        wait_num: int | None = None,
+        timeout: float | None = None,
+    ) -> None:
+        super().__init__(env_fns, DummyEnvWorker, wait_num, timeout)
```

Comment:
this is breaking. Have you made sure that **kwargs was never used? Pls edit the description of the PR to contain all breaking changes 

Pls also mention that users should refer to the docstring of the base class where appropriate

Reply:
Not sure I understand how this is breaking. The kwargs `wait_num` and `timeout` were already there and take on the same default values as in the base class. How could a `DummyVectorEnv` object be created that is not working anymore? By passing a dict with `wait_num` and `timeout` keys?

Reply:
It's breaking because it's a public interface, and previously the user could have passed any kwargs without an error being raised. Or am I wrong about this? I haven't checked how the kwargs were previous used

Reply:
Any other kwarg would throw an error because `BaseVectorEnv` only takes 4 arguments
https://github.com/thu-ml/tianshou/blob/1714c7f2c7295e58c9b94f6dcdc270da23f3d74c/tianshou/env/venvs.py#L76-L82